operator/internal/handlers/lokistack_create_or_update.go

```diff
@@ -123,7 +135,12 @@ func CreateOrUpdateLokiStack(
 			}
 		}
 
-		objStore.TLS = &storageoptions.TLSConfig{CA: cm.Name, Key: stack.Spec.Storage.TLS.CAKey}
+		caKey := stack.Spec.Storage.TLS.CAKey
+		if caKey == "" {
+			caKey = caKeyKey
+		}
```

Comment:
This works, but I think we have used the "initialize with default, override if set" style before:

```suggestion
		caKey := defaultCAKey 
		if stack.Spec.Storage.TLS.CAKey != "" {
			caKey = stack.Spec.Storage.TLS.CAKey
		}
```